frontend/src/controllers/IO.ts

```diff
@@ -575,9 +575,4 @@ class IO {
     }
 }
 
-const ioAddress =
-    process.env.NODE_ENV === "development"
-        ? "http://" + window.location.hostname + ":3001"
-        : "https://" + window.location.hostname;
-
-export default new IO(ioAddress);
```

Comment:
Removing this creates issues in development. Could we a proxy entry to `frontend/package.json`?

``` 
 "proxy": "http://localhost:3001",
```